data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -1137,7 +1137,6 @@ shop-wholesale                                      # icon z17- (also has captio
 === 500
 
 amenity-shower                                      # icon z17- (also has caption(optional) z17-)
-amenity-toilets                                     # icon z18- (also has caption(optional) z18-)
```

Comment:
I think toilets should be still above bbq (imagine a park with bbq places and a toilets block), outdoor seating, masts, silos and storage tanks still.

So either move them down too or just raise parkings. I think no one thought that there could be a parking with a toilets tag :)